torch/_ops.py

```diff
@@ -620,18 +687,14 @@ def handler(*args, **kwargs):
                             self, overload_types, args, kwargs
                         )
 
-                # Note [Not Caching Per-Dispatch-Key Mode Handlers]
```

Comment:
I would probably keep the note? (you reference it later on in the comments). But maybe rename to Not Caching PreDispatch modes.